app/domain/authentication/authn_jwt/vendor_configurations/configuration_factory.rb

```diff
@@ -0,0 +1,23 @@
+module Authentication
+  module AuthnJwt
+    module VendorConfigurations
+      # Factory that receives an authenticator name and returns the appropriate JWT vendor configuration class
+      class ConfigurationFactory
```

Comment:
can we make this a command class called `CreateJwtConfiguration`?

Reply:
Done